doc/releases/index.rst

```diff
@@ -128,5 +137,7 @@ Release notes and migration guides for end-of-life releases of Zephyr RTOS can b
 
 .. _`GitHub repository`: https://github.com/zephyrproject-rtos/zephyr
 .. _`GitHub tagged releases`: https://github.com/zephyrproject-rtos/zephyr/tags
+.. _`Zephyr 1.14.1 (LTS1)`: https://docs.zephyrproject.org/1.14.1/
+.. _`Zephyr 2.7.6 (LTS2)`: https://docs.zephyrproject.org/2.7.6/
 .. _`Zephyr 3.7.0`: https://docs.zephyrproject.org/3.7.0/
```

Comment:
```suggestion
.. _`Zephyr 3.7.0 (LTS3)`: https://docs.zephyrproject.org/3.7.0/
```

Reply:
yeah good point, also reordered the table so both have the most recent at the top, seems like a good idea